### PR TITLE
Updating to core 1614

### PR DESCRIPTION
**Pull request by andreasohlund**
Dependent on https://github.com/Particular/NServiceBus/pull/3561

### CODE REVIEW

**Comment by andreasohlund** [comment]
@Particular/container-maintainers please review

**Comment by WilliamBZA** [comment]
@andreasohlund tests are failing, but looks good otherwise.

**Comment by HEskandari** [comment]
There are four newly broken tests. Wonder how that happened as the develop branch seemed stable.

**Comment by andreasohlund** [comment]
Investigating the failures

**Comment by WilliamBZA** [comment]
Tests on `develop` still pass, it appears these are the new container tests that have changed in the past couple months.

Yet another example of how our feedback loop is too long between the container tests (and acceptance tests).

**Comment by andreasohlund** [comment]
All green now, one bug fixed and one core pull opened to deal with this https://github.com/Particular/NServiceBus/pull/3561

**Comment by WilliamBZA** [comment]
:+1: I've marked this as dependent on https://github.com/Particular/NServiceBus/pull/3561 so that we don't lose site of it.

@Particular/container-maintainers what is our process for cross-repository dependencies? Should we merge this or wait until the Core PR is merged in?

**Comment by andreasohlund** [comment]
I'd say that there is no hard dependency on 3561 so I'm :+1: to merge :smile:

**Comment by andreasohlund** [comment]
@Particular/container-maintainers https://github.com/Particular/NServiceBus/pull/3561 is now merged. Can we pull this one in?

**Comment by danielmarbach** [comment]
Good for me. 

The only question I had why we had previously `null` for the UoW lifecycle. Makes much more sense to have the Container lifecycle.

File: src/NServiceBus.StructureMap/StructureMapObjectBuilder.cs [hunk 1]
```diff
@@ -181,7 +181,7 @@ static ILifecycle GetLifecycleFrom(DependencyLifecycle dependencyLifecycle)
             case DependencyLifecycle.SingleInstance:
                 return new SingletonLifecycle();
             case DependencyLifecycle.InstancePerUnitOfWork:
-                return null;//null means the default lifecycle which is transient
+                return new ContainerLifecycle();//null means the default lifecycle which is transient
```
Comment (andreasohlund) [inline src/NServiceBus.StructureMap/StructureMapObjectBuilder.cs]: Duh, I see that I left the comment still there. I'll fix this

Comment (danielmarbach) [inline src/NServiceBus.StructureMap/StructureMapObjectBuilder.cs]: Yeah I was probably too fast to hit the merge button. My brain removed that comment.

**Pull request merged**